Source/Lib/Codec/EbSequenceControlSet.c

```diff
@@ -182,132 +182,15 @@ EB_ERRORTYPE CopySequenceControlSet(
-    dst->generalFrameOnlyConstraintFlag = src->generalFrameOnlyConstraintFlag;      writeCount += sizeof(EB_U32);
-    dst->targetBitrate              = src->targetBitrate;                           writeCount += sizeof(EB_U32);                     
-    dst->availableBandwidth         = src->availableBandwidth;                      writeCount += sizeof(EB_U32);                     
-    dst->qp                         = src->qp;                                      writeCount += sizeof(EB_U32);   
-    dst->enableQpScalingFlag        = src->enableQpScalingFlag;                     writeCount += sizeof(EB_BOOL);                  
-    dst->enableTmvpSps              = src->enableTmvpSps;                           writeCount += sizeof(EB_U32);   
-    dst->mvMergeTotalCount          = src->mvMergeTotalCount;                       writeCount += sizeof(EB_U32);            
-    dst->leftPadding                = src->leftPadding;                             writeCount += sizeof(EB_U16);
-    dst->rightPadding               = src->rightPadding;                            writeCount += sizeof(EB_U16);
-    dst->topPadding                 = src->topPadding;                              writeCount += sizeof(EB_U16);
-    dst->botPadding                 = src->botPadding;                              writeCount += sizeof(EB_U16);         
-    dst->enableDenoiseFlag          = src->enableDenoiseFlag;                       writeCount += sizeof(EB_BOOL);
-    dst->maxEncMode                 = src->maxEncMode;                              writeCount += sizeof(EB_U8);
-
-    // Segments
-    for (segmentIndex = 0; segmentIndex < MAX_TEMPORAL_LAYERS; ++segmentIndex) {
-        dst->meSegmentColumnCountArray[segmentIndex] = src->meSegmentColumnCountArray[segmentIndex]; writeCount += sizeof(EB_U32);
-        dst->meSegmentRowCountArray[segmentIndex] = src->meSegmentRowCountArray[segmentIndex]; writeCount += sizeof(EB_U32);
-        dst->encDecSegmentColCountArray[segmentIndex] = src->encDecSegmentColCountArray[segmentIndex]; writeCount += sizeof(EB_U32);
-        dst->encDecSegmentRowCountArray[segmentIndex] = src->encDecSegmentRowCountArray[segmentIndex]; writeCount += sizeof(EB_U32);
-    }
-
-    EB_MEMCPY(
-        &dst->bufferingPeriod,
-        &src->bufferingPeriod,
-        sizeof(AppBufferingPeriodSei_t));
-
-    writeCount += sizeof(AppBufferingPeriodSei_t);
-
-    EB_MEMCPY(
-        &dst->activeParameterSet,
-        &src->activeParameterSet,
-        sizeof(AppActiveparameterSetSei_t));
-
-    writeCount += sizeof(AppActiveparameterSetSei_t);
-
-    EB_MEMCPY(
-        &dst->recoveryPoint,
-        &src->recoveryPoint,
-        sizeof(AppRecoveryPoint_t));
-
-    writeCount += sizeof(AppRecoveryPoint_t);
+    AppVideoUsabilityInfo_t *videoUsabilityInfoPtr = dst->videoUsabilityInfoPtr;
+    size_t sizeCopy = (size_t)((EB_U64)(&dst->pictureControlSetPoolInitCount) -
+                               (EB_U64)(&dst->staticConfig));
```

Comment:
Hi Falei,

Some members of the SequenceControlSet_t object are missed to be copied, such as enableDenoiseFlag, maxEncMode, etc.

Reply:
Thanks for pointing out this issue. `enableDenoiseFlag`, `maxEncMode` and `activeParameterSet` were added in the new commit.